Makefile

```diff
@@ -4,6 +4,7 @@ test: lint install
 	# https://github.com/HumanCellAtlas/dcp-cli/issues/127
 	coverage run --source=hca -m unittest discover -v -t . -s test/upload
 	coverage run --source=hca -m unittest discover -v -t . -s test -p test_dss_*.py
+	coverage run --source=hca -m unittest discover -v -t . -s test -p test_fs_helper.py
```

Comment:
remove this change, please.

Reply:
How come?

Reply:
Oh got it. I'll merge https://github.com/HumanCellAtlas/dcp-cli/pull/143 first